scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketIO.java

```diff
@@ -493,11 +493,8 @@ public abstract static class TransformOutput<K1, K2, V> implements Serializable
 
   /** Represents a single sorted-bucket source written using {@link SortedBucketSink}. */
   public abstract static class Read<V> implements Serializable {
-
     @Nullable
-    abstract ImmutableList<String> getInputDirectories();
-
-    abstract String getFilenameSuffix();
```

Comment:
why has this been removed ?

Reply:
It's no longer a guarantee that a `SortedBucketIO.Read` implementation has a single `filenameSuffix` -- i.e. it could be both Parquet + Avro

I guess this could cause issues if we want to support Taps for SMB reads later down the line -- maybe the Builders could support an `ImmutableMap<String, String>` mapping input directories --> filename suffixes

Reply:
yeah 😅 I ended up adding a public getter for directories/FileOperations on `BucketedInput`, which should allow us to implement SMB taps later on